app/tests/spec/models/user.js

```diff
@@ -190,11 +191,12 @@ define(function (require, exports, module) {
 
     it('clearSignedInAccount', function () {
       sinon.spy(notifier, 'triggerRemote');
-      return user.setSignedInAccount({ email: 'email', uid: 'uid' })
+      const uid = createUid();
+      return user.setSignedInAccount({ email: 'email', uid })
```

Comment:
User account gets assigned `email: 'email'` and `email: 'testuser@testuser.com'`. It is hard for me to tell which/why it requires validation. Should you make all user objects use valid emails?